partner_private_information/models/res_partner.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    private_phone = fields.Char(string="Internal Phone")
+    private_mobile = fields.Char(string="Internal Mobile")
+    private_email = fields.Char(string="Internal Email")
+    private_notes = fields.Text(string="Internal Notes")
+    private_street = fields.Char(string="Internal Street")
+    private_street2 = fields.Char(string="Internal Street2")
+    private_zip = fields.Char(string="Internal Zip", change_default=True)
+    private_city = fields.Char(string="Internal City")
+    private_state_id = fields.Many2one(
+        "res.country.state",
+        string="Internal State",
+        ondelete="restrict",
+        domain="[('country_id', '=?', private_country_id)]",
+    )
+    private_country_id = fields.Many2one(
+        "res.country", string="Internal Country", ondelete="restrict"
+    )
+    private_country_code = fields.Char(
+        related="country_id.code", string="Internal Country Code"
+    )
+    private_category_ids = fields.Many2many(
+        "res.partner.category", "private_partner_category_rel", string="Internal Tags"
+    )
+
+    def _children_sync(self, values):
```

Comment:
Not the best way I think. If you deeply analise the function, you'll see it ends up calling _address_fields. That is the specific method you need to inherit.

Reply:
Done